.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: Build Android
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  install-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          yarn install
+      - name: Run tests
+        run: |
+          yarn test
+  build-android:
+    needs: install-and-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Install dependencies
+        run: |
+          yarn install
+      - name: Grant permissions to gradlew
+        run: |
+          cd android && chmod +x ./gradlew
+      - name: Generate Android Release APK
+        run: |
+          cd android && ./gradlew assembleRelease
```

Comment:
if you are creating a debug apk, this should not be `assembleRelease`

Reply:
@themakshter We need to assemble release build because when I used assembleDebug, the apk, when installed showed me the warning that the metro bundler is not running.

Reply:
I have created another step to create the debug